### DISCUSSION TITLE
My old Addon Revived

### DISCUSSION THREAD

**Discussion by VincentAraneo** [SuperTux/supertux#2765] [Show and tell]
I plan to revive an add-on I made in 2022. It was called The Snow, and it was a difficult add-on, but because I had to cancel it because I changed computers from an old one to a new one. So I'm going to relive it. Here's a teaser image (PD:The add-on this time will be for v0.7.0)

![teaser](https://github.com/SuperTux/supertux/assets/139823928/921324b1-f968-418f-bad2-562bb26d15ab)

**Comment by provigz** [comment]
Please use this repository as a whole for development-related issues, pull requests and discussions.
